packages/web-app-files/src/components/AppBar/ViewOptions.vue

```diff
@@ -1,17 +1,46 @@
 <template>
   <div class="oc-flex">
+    <template v-if="!hideViewOptions">
```

Comment:
Naming the whole component `ViewOptions` is a bit unlucky, but I'll rebase and introduce a different concept in #6662 so fine with me for now